lib/util/apiPaginate.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const LIMIT_DEFAULT = 50;
+const LIMIT_MAX = 1000;
+
+const OFFSET_DEFAULT = 0;
+
+const DEFAULT_MAX_RESULT_WINDOW = 10000;
```

Comment:
I think that this constant value should be config, but currently not 🙇

Reply:
Okay for now 👍